sdktests/server_side_big_segments.go

```diff
@@ -0,0 +1,502 @@
+package sdktests
+
```

Comment:
This file is pretty big— sorry— I tried to factor out boilerplate where I could, but I could probably do more. But I hope the structure is clear at least.